serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -33,53 +26,59 @@ If using Maven, this means including following snippet in pom.xml:
 </dependency>
 ----
 
-== Configuring Smallrye channels
+The messaging capabilities are included in the Quarkus Serverless Workflow extension, even though the messaging capabilities are optional. This means you do not need to explicitly add the messaging add-on dependency when using Serverless Workflow. However, you need to add the Kafka Quarkus Smallrye connector dependency to indicate that you are using Apache Kafka.
 
-This section explains how to configure Smallrye channels for a Kogito Serverless Workflow using event definitions. This info is extracted from link:{kogito_messaging_url}[Kogito documentation]
+[[con-sw-smallrye-channel-configuration]]
+== Smallrye channels configuration for a Serverless Workflow
 
-Channels are defined by using link:{quarkus_config_url}[Quarkus configuration]. The pattern for channel properties is `mp.messaging.[incoming|outgoing].<channel name>.<property_name>`.
+You can configure Smallrye channels for a Severless Workflow using event definitions. The Smallrye channels are defined using link:{quarkus_config_url}[Quarkus configuration]. The format for Smallrye channel properties is `mp.messaging.[incoming|outgoing].<channel name>.<property_name>`. 
 
-Kogito serverless workflow allows different channel mapping strategies:
+Serverless Workflow allows the following channel mapping strategies:
 
 * Define one default incoming channel to receive all the incoming messages and one default outgoing channel to store all the published messages.
+* Define a channel for each link:{cloud_events_url}[CloudEvent] type so that every message type has a dedicated channel.
+* Define a channel for certain CloudEvent types. Allow the non-mapped CloudEvent types to use the default incoming or outgoing channel.
```

Comment:
Remove allow. 
The sentence should read
"Define a channel for certain cloudevent type. The non mapped cloudevent types uses default channel.

Reply:
Done